diesel/src/mysql/connection/bind.rs

```diff
@@ -9,6 +9,7 @@ pub struct Binds {
     lengths: Vec<libc::c_ulong>,
     is_nulls: Vec<ffi::my_bool>,
     mysql_binds: Vec<ffi::MYSQL_BIND>,
+    errors: Option<Vec<ffi::my_bool>>,
```

Comment:
Hm interesting point. If nothing else, `mysql_binds` will always need to be in its own vec.